common/log.c

```diff
@@ -592,7 +592,7 @@ log_message(const enum logLevels lvl, const char *msg, ...)
         /* log to syslog*/
         /* %s fix compiler warning 'not a string literal' */
         syslog(internal_log_xrdp2syslog(lvl), "(%d)(%ld)%s", g_getpid(),
-               tc_get_threadid(), buff + 20);
+               (long) tc_get_threadid(), buff + 20);
```

Comment:
same here,
Lets not change this now as it's always big enough and this PR is just for warnings.